src/components/details-view/details-view.js

```diff
@@ -352,7 +352,10 @@ class DetailsView extends Component {
                     onToggle={onListToggle}
                     listType={listType}
                   >
-                    {isListAccordionOpen ? renderList(isSticky) : null}
+                    {isListAccordionOpen
+                      ? renderList(isSticky)
+                      : null
+                    }
```

Comment:
Could it be changed by
```suggestion
                    {isListAccordionOpen && renderList(isSticky)}
```
?

Reply:
Sure, fixed it, thanks